src/test/java/core/basesyntax/service/FruitServiceTest.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax.service;
+
+import static org.junit.Assert.assertNotNull;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.impl.CsvFileReaderImpl;
+import core.basesyntax.service.impl.CsvParserImpl;
+import core.basesyntax.service.impl.FruitServiceImpl;
+import java.util.List;
+import org.junit.Before;
+import org.junit.Test;
+
+public class FruitServiceTest {
+    private static final String FILE_FROM = "src/test/resources/shop_activity_test.csv";
+    private static final String FILE_INVALID_OPERATION
+            = "src/test/resources/shop_activity_test_invalid_operation.csv";
+    private List<String> data;
+    private FruitService service;
+
+    @Before
+    public void setUp() {
+        service = new FruitServiceImpl();
+    }
+
+    @Test
+    public void validFileData() {
+        data = new CsvFileReaderImpl().readFromFile(FILE_FROM);
+        List<String[]> parsedData = new CsvParserImpl().parse(data);
```

Comment:
let's check parser separately and do not depend on the other services here

pass some hardcoded values to the method and then check the storage - we should not read from a file here